go.mod

```diff
@@ -5,7 +5,7 @@ go 1.19
 require (
 	crawshaw.io/sqlite v0.3.3-0.20211227050848-2cdb5c1a86a1
 	github.com/ALTree/bigfloat v0.0.0-20220102081255-38c8b72a9924
-	github.com/chaos-mesh/chaos-mesh/api v0.0.0-20221213145053-386291e73746
```

Comment:
i'd really prefer to remove unnecessary files from this PR. go.mod and go.sum

Reply:
The versions used to build `go-spacemesh` are the ones in this `go.mod` file even if I do not update it. Dependencies are overwritten if a dependent package uses a newer version of a package listed here.

These are updates from `go work sync` the only thing it does is to synchronize dependencies between our repositories, it does not update.

Reply:
i still don't understand the relevance. are you talking about indirect dependencies?
which other repository does this PR require to sync with?

Reply:
what I mean is we are already using newer versions of dependencies in other packages. I only did `go work sync` to align the versions of dependencies we use between all of our codebase.

When compiling the code it will not necessarily use what's in `go.mod` but might overwrite that dependency when another package uses a newer version of it. `go work sync` just makes visible what actually is compiled taking all (also indirect) dependencies into account.

I reverted this file now and will update again in a new PR 🙂